File: 1st/Hyeseung/Programmers/영어가 싫어요.java
class Solution {
    public long solution(String numbers) {
        String number = numbers
            .replace("zero", "0")
            .replace("one", "1")
            .replace("two", "2") 
            .replace("three", "3")
            .replace("four", "4")
            .replace("five", "5")
            .replace("six", "6")
            .replace("seven", "7")
            .replace("eight", "8")
            .replace("nine", "9");
        return  Long.parseLong(number);
    }
}

File: 2st/Yeonwoo/영어가 싫어요.py
def solution(numbers):
    answer = 0
    integer_list = ["zero","one", "two", "three", "four", "five", "six", "seven","eight", "nine"]
    for i, num in enumerate(integer_list):
        numbers = numbers.replace(num,str(i))
    answer= int(numbers)
    return answer